src/main/scala/org/broadinstitute/dsde/workbench/sam/api/SamRoutes.scala

```diff
@@ -48,6 +48,7 @@ abstract class SamRoutes(val resourceService: ResourceService, val userService:
         case Complete(resp) =>
           val logLevel: LogLevel = resp.status.intValue / 100 match {
             case 5 => Logging.ErrorLevel
+            case 4 => Logging.InfoLevel
```

Comment:
unrelated change that I made to rawls to allow us to select to log 4xx level errors without swamping the logs with successful responses too